tests/store/metadata/oxigraph/test_oxigraph_versions.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from pydantic import ValidationError
+
+from store.metadata.oxigraph.store import OxigraphMetadataStore
+import schemas
+
+
+def test_oxigraph_get_versions_returns_valid_structure():
+    """
+    Confirm that the OxigraphMetadataStore.get_versions()
+    function returns a versions that matches the Versions
+    schema.
+    """
+    store = OxigraphMetadataStore()
+
+    versions = store.get_versions("4gc", "2023-09")
+    versions_schema = schemas.Versions(**versions)
+    assert versions_schema.context == "https://staging.idpd.uk/ns#"
+    assert versions_schema.id == "https://staging.idpd.uk/datasets/4gc/editions/2023-09/versions"
+    assert versions_schema.type == "hydra:Collection"
+    assert versions_schema.title == "4G Coverage"
+    assert len(versions_schema.versions) > 0
+    assert versions_schema.count == 1
```

Comment:
best not check this, the tests will start failing if someone adds a version 2. You could assert it matches len(versions_schema.versions) if you like.